DuggaSys/gitfetchService.php

```diff
@@ -252,7 +219,6 @@ function bfs($url, $cid, $opt)
                                         array_push($filesVisited,$item['name']);
                                     }
                                 }
-                                // Otherwise, fetch and download all files
                             } else {
                                 if (!in_array($item['name'], $filesVisited)) {
                                     if ($opt == "REFRESH") {
```

Comment:
Removing comments makes it harder to understand the code. If the comments are not accurate, make new ones.